<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{border-image: url(:/pig/huaPig.png);}
</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:20px
}</string>
        </property>
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:20px
}</string>
        </property>
        <property name="text">
         <string>密    码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPswd"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnRegister">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>确认注册</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
